main.py

```diff
@@ -220,11 +220,15 @@ def vms(self):
         vms = []
 
         for line in out.splitlines():
-            m = re.search(r'"([a-zA-Z0-9-_]+)"', line)
-            if m:
-                name = m.group(1)
+            m = re.findall(r'([a-zA-Z0-9-_]+)', line)
```

Comment:
In the first written regex there were double quote characters that doesn't exist in your PR, is it on purpose?

Reply:
without the qoutes:
![image](https://user-images.githubusercontent.com/35425887/59976647-41221680-95d0-11e9-894b-f662d53019cf.png)


with the qoutes:
![image](https://user-images.githubusercontent.com/35425887/59976651-4da66f00-95d0-11e9-823e-f1dcb6b960ac.png)

Reply:
so this is indeed with a purpose :)